setup/setup.sh

```diff
@@ -26,6 +26,10 @@ if [[ `grep "Kali GNU/Linux.*\(2\|Rolling\)" /etc/issue` ]]; then
   osinfo="Kali2"
 fi
 
+# make sure we run from this directory
+pushd . > /dev/null
+cd "$(dirname "$0")"
```

Comment:
Line 30 and 31 could be combined to a single pushd command

Reply:
@paalbra I've updated the setup script as per your suggestion.